src/nodes.coffee

```diff
@@ -1695,14 +1697,18 @@ exports.Code = class Code extends Base
     # parameters need to be sliced off the end of the splat parameter’s
     # array of arguments passed into the function.
     if paramsAfterSplat isnt 0
-      ref = new IdentifierLiteral splatParamName
-      val = new Call (new Value ref, [new Access new PropertyName 'slice']), [
-        new NumberLiteral 0
-        (new Op '-',
-          new Value(new IdentifierLiteral(splatParamName), [new Access new PropertyName 'length']),
-          new NumberLiteral paramsAfterSplat)
-      ]
-      exprs.push new Assign ref, val, '='
+      exprs.push new Assign new IdentifierLiteral(splatParamName), new Call (
+                                                                     new Value new IdentifierLiteral(splatParamName),
+                                                                     [new Access new PropertyName 'slice']),
+                                                                     [
+                                                                        new NumberLiteral 0
+                                                                        (new Op '-',
+                                                                          new Value(
+                                                                            new IdentifierLiteral(splatParamName),
+                                                                            [new Access new PropertyName 'length']
+                                                                          ),
+                                                                          new NumberLiteral paramsAfterSplat)
+                                                                     ]
```

Comment:
I was following the style of grammar.coffee. I'm happy to pull it back left though, that would be my preference; I just thought this was the intended style.